__tests__/app.test.js

```diff
@@ -91,3 +91,27 @@ describe('GET /api/articles/:article_id', () => {
 			});
 	});
 });
+
+describe('GET /api/articles/', () => {
+	test('200: return all articles', () => {
+		return request(app)
+			.get(`/api/articles/`)
+			.expect(200)
+			.then(({ body }) => {
+				const { articles } = body;
+				console.log(typeof articles);
+				for (const article of articles) {
+					expect(article).toMatchObject({
+						article_id: expect.any(Number),
+						title: expect.any(String),
+						topic: expect.any(String),
+						author: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+						comment_count: expect.any(Number),
+					});
+				}
+			});
+	});
```

Comment:
I would like to see another test that checks whether the articles are sorted based on their created_at property - have a look at the jest-sorted library to help with this